ament_copyright/ament_copyright/parser.py

```diff
@@ -93,8 +89,7 @@ def __init__(self, path):
 
         self.copyright_identifiers = []
 
-    def identify_copyright(self):
-        known_copyrights = get_copyright_names()
+    def identify_copyright(self, known_copyrights):
```

Comment:
If I am understanding this PR correctly the crux of the change is the redundant call to `get_copyright_names`. This revision kicks that call up to the main function. 

Is that correct?

Reply:
Yes, exactly.